addons/web_editor/static/lib/odoo-editor/src/commands/commands.js

```diff
@@ -206,7 +205,15 @@ function align(editor, mode) {
             if (!visitedBlocks.has(block)) {
                 const hasModifier = getComputedStyle(block).textAlign === mode;
                 if (!hasModifier && block.isContentEditable) {
-                    block.oAlign(sel.anchorOffset, mode);
+                    block.oAlign(mode);
+                    const link = closestElement(node, 'a');
+                    if (link) {
```

Comment:
So... I guess this is why the other code above just ignores `A` tags ? Because you'll handle it manually here ? Wouldn't it be possible to detect the case here and _not_ call `oAlign` in that case so that we could remove the ad-hoc case in `oAlign` ? Just an suggestion, it's no big deal if it wouldn't work.

Reply:
That would involve duplicating the code inside `oAlign`, unless you're seeing something I'm not.

Reply:
I just realized I misread the code in `oAlign`. I thought you skipped the function for links but that's actually the opposite. You want to run the function even though links are not blocks. But then I don't understand why we need to manually call it _again_ below.

Reply:
That's the whole idea of the commit: we call it on the block but for links we _also_ call it on the link itself so the alignment of the link matches that of its parent block. Anyway I'm making a proposal for another way to fix the bug without actually changing the alignment. I'll let you know when it's done.